components/shared/layout/sidebar.tsx

```diff
@@ -13,11 +13,11 @@ import { cn } from "@/lib/utils";
 import { Button } from "@/components/ui/button";
 
 const menuItems = [
```

Comment:
make it separate the menu and import it. we can manage this from one place. we have to do it, as easy to access